benchmarks/gpt_fast/benchmark.py

```diff
@@ -16,14 +21,17 @@ class Experiment:
     metric: str
     target: float
     actual: float
+    model: str
+    quantization: str
+    device: str
```

Comment:
I think we don't need ```model```, actually ```name``` is model's name. It's better to rename ```quantization``` to ```dtype``` and remove the dtype suffix from name.

Reply:
Sounds good, let's stick with the model name from huggingface because I assume that the name most people will be familiar with.  It will then show up in the same way on the dashboard.